soh/src/code/audio_load.c

```diff
@@ -1346,7 +1346,9 @@ void AudioLoad_Init(void* heap, size_t heapSize) {
     int customSeqListSize = 0;
     char** seqList = ResourceMgr_ListFiles("audio/sequences*", &seqListSize);
     char** customSeqList = ResourceMgr_ListFiles("custom/music/*", &customSeqListSize);
-    sequenceMapSize = (size_t)(seqListSize + customSeqListSize);
+    // 20 is arbitrary and meant as only a temporary fix. this just gives headroom for the sequences in AudioCollection that take up early indexes that
```

Comment:
we should probably figure out what the actual number is, and then `#define` it and give it a nice name

Reply:
There were six within the rang of DJ's custom sequence count, but there were also other ones in the low 2000, mid 6000, and 10-12k ranges, too. I really don't know what is going on in `AudioCollection` with all of those non-sequence entries.

Reply:
I did figure out that I could do an allocation for `audio_load`'s sequenceMap based on the size of `AudioCollection`'s sequenceMap at that point in the load since the other non-sequence audio assets are already loaded into `AudioCollection`, so that can get rid of this part, at least.